src/compose/ComposeBox.js

```diff
@@ -63,7 +62,7 @@ type Props = $ReadOnly<{|
   insets: EdgeInsets,
 
   narrow: Narrow,
-  editMessage: EditMessage | null,
+  editing: boolean,
```

Comment:
nit: I might name this `isEditing`, for more explicitness that it's a boolean

Reply:
Renamed :)